### DISCUSSION TITLE
Tomcat how to deploy multiple services?

### DISCUSSION THREAD

**Discussion by wangliguang517** [apache/skywalking#9373] [General]
If tomcat deploy multiple services.
How can i set up ”agent.service_name“.

Now ,I set up ”agent.service_name=service1,service2,servie3“.
But skywalking's ui can not distinguish the request of three services。

**Comment by wu-sheng** [comment]
This is a duplicated one. We announced a long time ago, that this is not supported. 

SkyWalking: an APM(application performance monitor) system, **especially designed for microservices**, cloud native and container-based architectures.

No one in microservice architecture would use this way to deploy.

**Reply by wangliguang517** [reply]
If skywalking can support in the old version?
Beacase company have a lot of tomcat standalone deployment in the current.

I find someone have same problem.
He set up "agent.service_name=test1,agent.service_name=test2".
https://blog.csdn.net/JavaAlpha/article/details/121538055
But I don't konw if it's right.

**Reply by wu-sheng** [reply]
No, micro-service is why we started SkyWalking. This is never supported.

**Reply by wangliguang517** [reply]
Thank you.
谢谢，同时很高兴，一睹大牛风采。
